api/src/main/java/org/jenkinsci/plugins/workflow/graph/FlowNode.java

```diff
@@ -141,6 +141,20 @@ public String getDisplayName() {
         else            return getTypeDisplayName();
     }
 
+    public String getDisplayFunctionName() {
+        String functionName = getTypeFunctionName();
+        if (functionName == null) {
+            return getDisplayName();
```

Comment:
Why not make `getTypeFunctionName` return `@Nonnull` and default to `getDisplayName`?